### DISCUSSION TITLE
m62 - 65 not woking

### DISCUSSION THREAD

**Discussion by jeefdumbo** [grblHAL/iMXRT1062#86] [General]
I build a firmware with WebBuilder for  T41U5XBB Board with 4 Axis and Y Autosquared
When i Control the Aux outputs with M62P0 or M64P0 M63P0 or M65P0 i get the error 36:  unused value words found in block
How can i Turn on or Off the Aux Ports?

**Comment by terjeio** [comment]
It is a regression, will commit a fix tomorrow.
